### ISSUE TITLE
/camera/cameraAction!findAllDevices.action not compatible with shared account

### ISSUE THREAD

**Issue by BaQs** [BaQs/pyEzviz#34]
I use a specific, shared account for my home automation.
this endpoint returns no device.

But works with the main account.

I would recommend using pagelist only for devices status.

what do you think @RenierM26 ?

**Comment by BaQs** [comment]
could you please review branch https://github.com/BaQs/pyEzviz/compare/ get_device?expand=1 ?

**Comment by RenierM26** [comment]
Hi BaQs, 

Only the following attributes still used that method:

Defence enabled/disabled
Defence schedule enabled/disabled
Device MAC adress.

I found the defence items hiden away in pagelists. I have removed the deviceinfo method completely...just can't find device mac address so I've just disabled this one attribute for now. 

Would you mind testing with your shared account?

**Comment by RenierM26** [comment]
The only ajax/jquery api call left is the one used to enable/disable motion detection (defence). Once I've found the API alternative, I'll replace that one as well.

It's very difficult to capture https traffic from android as they use "certificate pinning" in the app......I finally found a utility to override this (frida). I can now use fidler(windows) to capture all app traffic. 
(I only have android and non-apple computers.)

Guess android is more secure than apple ;-)
